### DISCUSSION TITLE
Workflows to update the default list of opaque ports for CNCF projects

### DISCUSSION THREAD

**Discussion by guhuajun** [linkerd/linkerd2#8439] [Ideas] [answered]
Greetings,

I found my NATS traffic was blocked by linkerd as _a protocol detection timeout_, I have documented my findings in one discussion of nats-py project, https://github.com/nats-io/nats.py/discussions/307

For specific Linkerd version, it's stable-2.11.0.

From linkerd project perspective, will it be possible to provide an official workflow to update to update the default list of opaque ports for CNCF projects. Or will linkerd project setup a formal validation process for CNCF projects?

Thanks,
Greg

**Comment by olix0r** [comment, marked as answer]
Duplicate of https://github.com/linkerd/linkerd2/discussions/8438
